backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -273,4 +280,56 @@ public static Extension convertToTribalAffiliationExtension(String tribalAffilia
     }
     return null;
   }
+
+  public static Device convertToDevice(DeviceType deviceType) {
+    if (deviceType != null) {
+      return convertToDevice(deviceType.getManufacturer(), deviceType.getModel());
+    }
+    return null;
+  }
+
+  public static Device convertToDevice(String manufacturer, String model) {
+    if (StringUtils.isNotBlank(manufacturer) || StringUtils.isNotBlank(model)) {
+      return new Device()
+          .setManufacturer(manufacturer)
+          .addDeviceName(
+              new DeviceDeviceNameComponent()
+                  .setName(model)
+                  .setType(DeviceNameType.MANUFACTURERNAME));
```

Comment:
😊 I think you are right! When I read this the first time I thought manufacturer name was the name it was given by manufacturer rather than the actual name of the manufacturer.